docs/Configuration.md

```diff
@@ -228,7 +228,7 @@ Here is a list of all features which can be toggled on/off and their default val
 | `ConsiderPatternsForDirectTransfers` | Enable limiting transfers so that there is only a single transfer to each pattern.                                                                                                                        |         ✓️         |         |
 | `DebugClient`                        | Enable the debug web client located at the root of the web server.                                                                                                                                        |         ✓️         |         |
 | `FloatingBike`                       | Enable floating bike routing.                                                                                                                                                                             |         ✓️         |         |
-| `GtfsGraphQlApi`                     | Enable GTFS GraphQL API.                                                                                                                                                                                  |         ✓️         |    ✓️   |
+| `GtfsGraphQlApi`                     | Enable GTFS GraphQL API.                                                                                                                                                                                  |         ✓️         |         |
```

Comment:
Btw what do we want to do in the long run with these APIs. I guess it's reasonable to allow someone to disable certain APIs to minimize security issues or because they don't want API user feedback regarding some API they are not familiar with. On the other hand, having some API disabled on an OTP instance makes the developer experience worse if a client is written to work against certain API that is not necessarily always enabled.

Reply:
IIRC then we said that it will be possible to turn off all APIs via feature toggles and the GraphQL ones will be enabled by default.